scripts/table_updates/config.json

```diff
@@ -18,5 +18,8 @@
         "NSCLC2": "syn51318736",
         "CRC2": "syn52943210",
         "RENAL": "syn59474249"
-    }
+    },
+    "main_genie_release_version": "16.6-consortium",
+    "main_genie_data_release_files": "syn16804261",
```

Comment:
Nit: Does it make sense to use the database to synapse id mapping table?  (Maybe in the future - but not for this iteration, if we are going to re-design the whole process)

Reply:
I think so for future iterations at least for retrieving the specific tables yes. I just wanted to limit dependencies for this one.